cookbooks/bcpc-hadoop/recipes/hdfsdu_pre_fetch.rb

```diff
@@ -0,0 +1,24 @@
+# Setup HDFSDU config
+
+set_hosts
+
+@rm_hosts = node[:bcpc][:hadoop][:rm_hosts]
+if defined?@rm_hosts and not @rm_hosts.empty? then
+   node.override[:hdfsdu][:jobtracker] = "#{float_host(@rm_hosts.first[:hostname])}:8032" 
+end
+
+node.override[:hdfsdu][:namenode] = "hdfs://#{node.chef_environment}"
```

Comment:
Changed code to use `node.bcpc.hadoop.hdfs_url`. However, I had to remove the trailing `/` from the value for oozie to be happy. Its a separate commit in this PR.